docker-compose-java-development.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.5'
+services:
+
+  java-development:
+    image: eventuateio/eventuate-tram-sagas-examples-customers-and-orders-java-development:latest
+    volumes:
+      - .:/eventuate-tram-sagas-examples-customers-and-orders-java-development
+      - /var/run/docker.sock:/var/run/docker.sock
+    environment:
+      SPRING_DATASOURCE_URL: jdbc:mysql://mysql/eventuate
+      SPRING_DATASOURCE_USERNAME: mysqluser
+      SPRING_DATASOURCE_PASSWORD: mysqlpw
+      SPRING_DATASOURCE_DRIVER_CLASS_NAME: com.mysql.jdbc.Driver
+      EVENTUATELOCAL_KAFKA_BOOTSTRAP_SERVERS: kafka:29092
+      EVENTUATELOCAL_ZOOKEEPER_CONNECTION_STRING: zookeeper:2181
+      SPRING_SLEUTH_ENABLED: "true"
```

Comment:
java-development is not a service - it doesn't not need anything related to Sleuth!

Reply:
ok